.buildkite/scripts/unit_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+source .buildkite/scripts/common.sh
+
+set -euo pipefail
+
+beats_subfolder=$1
+
+sudo chmod -R go-w ${beats_subfolder}/
+
+echo "--- Run Unit Tests"
+pushd "${beats_subfolder}" > /dev/null
+
+umask 0022
```

Comment:
can we add a small `#TODO remove when custom vm images are used` as we have fixed this e.g. in the Logstash base images https://docs.google.com/document/d/1IDM9FEyrlrKmjZge3iBa-maaOFeK_ONjxLBMEXkwLI0/edit#bookmark=id.x7640ku6nmd3

(I assume this script will run on Linux, I am not sure if we need to also cover for macOS, in which case, custom images won't help, as we can't have custom images yet for the macos provider)